<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabDatos">
     <attribute name="title">
      <string>Datos</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>198</width>
        <height>342</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Dato x1</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Dato x2</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_2</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Dato x3</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_3</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Dato x4</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_4</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Dato x5</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_5</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Dato x6</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_6</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Dato x7</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_7</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Dato x8</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_8</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Dato x9</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_9</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Dato x10</string>
               </property>
               <property name="buddy">
                <cstring>spinBox_10</cstring>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QSpinBox" name="spinBox"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_2"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_3"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_4"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_5"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_6"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_7"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_8"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_9"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_10"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Dato y1</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Dato y2</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Dato y3</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Dato y4</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Dato y5</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Dato y6</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Dato y7</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Dato y8</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Dato y9</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Dato y10</string>
               </property>
               <property name="buddy">
                <cstring>spinBox</cstring>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QSpinBox" name="spinBox_11"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_12"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_13"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_14"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_15"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_16"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_17"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_18"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_19"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_20"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Generar Gráfica</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Generar Informe</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>441</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInforme">
     <attribute name="title">
      <string>Informe</string>
     </attribute>
     <widget class="QListView" name="listView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>601</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
